README.md

```diff
@@ -535,7 +535,10 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 - [Türkçe Python Kaynakları](https://github.com/ibrahimirdem/turkce-python-kaynaklari)
 - [Python Programlama Dili](https://medium.com/@halilozel1903/python-programlama-dili-1f1d88ef1e1d) - [Halil ÖZEL](https://github.com/halilozel1903)
 - :movie_camera: [6 Saatlik Python Eğitimi](https://www.youtube.com/watch?v=6IWpowC2ryc) - [Engin Demiroğ](https://www.youtube.com/channel/UCRjiquPh4mjPNoOV9eCilXQ)
-
+- :movie_camera: [Yakın Kampüs](https://www.youtube.com/watch?v=EzHgbO1Cee4&list=PLWctyKyPphPiul3WbHkniANLqSheBVP3O&ab_channel=Yak%C4%B1nKamp%C3%BCs)
+- [Merve Noyan - NLP](https://merveenoyan.medium.com/do%C4%9Fal-dil-i%CC%87%C5%9Fleme-natural-language-processing-2d7c72daf245)
```

Comment:
Format, `Baslik - Yazar` seklinde. Merve Noyan'la bi sorunum yok ama neden ozellikle bu sekilde baslama geregi duydunuz?

Reply:
Onun yazdığı konu olduğu için o şekilde yazdım. Kursların yanında genellikle kursu veren kişilerin ismi yazıyordu ondan belirtmek istedim. Hemen düzenliyorum.